recipes/bob.extension/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+PYTHONDONTWRITEBYTECODE=1 $PYTHON setup.py install
```

Comment:
Also, setting `PYTHONDONTWRITEBYTECODE` should be the same as passing option [`-B`]( https://docs.python.org/2/using/cmdline.html#miscellaneous-options ). Could we please do that instead?